tests/metadata_stores/_ducklake_utils.py

```diff
@@ -0,0 +1,18 @@
+"""Test helpers for DuckLake-enabled scenarios."""
+
+from __future__ import annotations
+
+import duckdb
+import pytest
+
+
+def ensure_ducklake_extension() -> None:
```

Comment:
this shouldn't be needed, tests must error our if the extension was not installed for some reason

Reply:
removed